validators/types_unix.go

```diff
@@ -66,10 +67,10 @@ var DefaultSysSpec = SysSpec{
 	CgroupsOptional: []string{
 		// The hugetlb cgroup is optional since some kernels are compiled without support for huge pages
 		// and therefore lacks corresponding hugetlb cgroup
-		"hugetlb",
+		"hugetlb", "blkio",
```

Comment:
we should have this on a separate line and have a description comment similarly to the one for hugetlb.

Reply:
Add some description now.